### PR TITLE
(doc) Update press release

### PR DESCRIPTION
**Pull request by cgrinaldi**

### CODE REVIEW

**Comment by garrettjoecox** [comment]
Looks great!

**Comment by cgrinaldi** [comment]
Thanks!

File: _PRESS-RELEASE.md [hunk 1]
```diff
@@ -9,36 +7,24 @@ For new initiatives a product manager typically starts by writing an internal pr
...
 
-## Quote from You ##
-  > A quote from a spokesperson in your company.
+## Summary
+**Recall.js** will help keep your technical skills up-to-date and prepare you to take on a technical interview at a moment’s notice. It uses a spaced-repetition learning algorithm to optimize your recall of technical interview questions. In addition, its focus on TDD and in-browser coding provides a productive learning environment for evaluating your performance on programming challenges.
 
-## How to Get Started ##
-  > Describe how easy it is to get started.
+## Problem
+Practicing technical interview questions is incredibly important when preparing for a software engineering interview. When an individual decides to look for another position, he/she must spend time preparing for such questions. Often times, this individual reviews coding challenges that they have practiced for prior interviews. Not only is this time wasted, but it reflects the degradation of engineering knowledge over time.
```
Comment (mosse) [inline _PRESS-RELEASE.md:25]: Practising

Comment (mosse) [inline _PRESS-RELEASE.md:25]: I am a wrong.

**Pull request merged**